src/foam/nanos/pii/PIIReport.flow

```diff
@@ -0,0 +1,46 @@
+<h1>Personal Identifiable Information (PII)</h1>
```

Comment:
Add a <title> field, and my new PR will pick it up for the menu

Reply:
updated